client/room.html

```diff
@@ -7,8 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0">
   <link rel="icon" type="image/svg+xml" href="/i/branding/favicon.svg"/>
   <link rel="alternate icon" href="/i/branding/favicon.png"/>
-  <link rel="preconnect" href="https://fonts.gstatic.com">
-  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  <link href="/i/fonts/fonts.css" rel="stylesheet">
```

Comment:
This can be removed now, right?

Reply:
It's still there and I don't know why. It doesn't even load because it's served as text/html.